config/Migrations/20230102000001_MigrationQueueReplaceFailedWithAttempted.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+use Phinx\Migration\AbstractMigration;
+
+class MigrationQueueReplaceFailedWithAttempted extends AbstractMigration {
+
+	/**
+	 * Change Method.
+	 *
+	 * Write your reversible migrations using this method.
+	 *
+	 * More information on writing migrations is available here:
+	 * http://docs.phinx.org/en/latest/migrations.html#the-abstractmigration-class
+	 *
+	 * @return void
+	 */
+	public function change() {
+		$this->table('queued_jobs')
+			->renameColumn('failed', 'attempts')
+			->update();
+		$this->query('UPDATE queued_jobs SET attempts = attempts + 1 WHERE completed IS NOT NULL');
```

Comment:
Do we need this?
We can assume tasks not run are attempt 0 still

Reply:
I don't quite follow?

I figured that jobs which _are_ completed (`completed IS NOT NULL`), need to have attempts incremented by 1.

A job may have completed, and have _not_ failed, and since we rename failed->attempts, it would appear to have 0 attempts, which I think is misleading. It should have at least 1 attempt, as it is completed.

Reply:
True